integration-cli/test_vars_unix.go

```diff
@@ -0,0 +1,8 @@
+// +build !windows
```

Comment:
Just following the conversation from rest of the code base. This is probably fine and if it's not it's will be a compilation error which can be easily fixed.